arbitrator/config/arbitrator_params.yaml

```diff
@@ -26,4 +26,4 @@ use_fixed_costs: true
 # Map: The priorities associated with each plugin during the planning 
 # process, values will be normalized at runtime and inverted into costs
 # Unit: N/a
-plugin_priorities: {RouteFollowingPlugin: 4.0, WorkZonePlugin: 8.0, PlatooningStrategicPlugin: 1.0}
+plugin_priorities: {SCIStrategicPlugin: 4.0, RouteFollowingPlugin: 2.0, WorkZonePlugin: 1.0, PlatooningStrategicPlugin: 1.0}
```

Comment:
This change currently disables the workzone plugin. Is that really necessary? Since it can only be triggered via a highly specific carma-cloud message it seems like it could be safely kept at a higher priority than RouteFollowingPlugin

Reply:
never mind, github caching didn't show this correctly on my end.